2.6/get-pip.py

```diff
@@ -135,7 +135,7 @@ def parse_args(self, args):
     for arg in args:
         try:
             req = InstallRequirement.from_line(arg)
-        except:
+        except Exception:
```

Comment:
Unrelated change?

Reply:
It's related, cherry-picked from #23. Some of linters (flake8) was yelling at it. I think, adding linters should go along with cleaned code.

Reply:
If they are fixes for linter issues, can they be put in a separate PR, please?

Reply:
@pfmoore Done!

Reply:
#30